models/classes/runner/synchronisation/action/Move.php

```diff
@@ -75,7 +75,6 @@ public function process()
                 $this->setOffline();
             }
 
-            $serviceContext->getTestSession()->initItemTimer($this->getTime());
```

Comment:
The purpose of this call is described in this PR, I assume it should remain here: https://github.com/oat-sa/extension-tao-testqti/pull/559